light-client/relayer/src/bin/main.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2018-2023 The MobileCoin Foundation
+
+//! Main Method for the light client relayer
+
+use mc_common::{logger, sentry};
+use mc_ledger_db::LedgerDB;
+use mc_light_client_relayer::{Config, DummySender, Relayer};
+use mc_util_cli::ParserWithBuildInfo;
+use mc_util_grpc::AdminServer;
+use mc_watcher::watcher_db::WatcherDB;
+use std::sync::Arc;
+
+fn main() {
+    std::env::set_var("MC_LOG_STDERR", "1");
+    let (logger, _global_logger_guard) = logger::create_app_logger(logger::o!());
+
+    mc_common::setup_panic_handler();
+    let _sentry_guard = sentry::init();
+
+    let config = Config::parse();
+
+    let config_json = serde_json::to_string(&config).expect("failed to serialize config to JSON");
+    let get_config_json = Arc::new(move || Ok(config_json.clone()));
+    let _admin_server = config.admin_listen_uri.as_ref().map(|admin_listen_uri| {
+        AdminServer::start(
+            None,
+            admin_listen_uri,
+            "Light Client Relayer".to_owned(),
+            "".to_string(), //config.client_listen_uri.to_string(),
```

Comment:
:wrench: It may be that this will be needed later, but having it commented out with no explanation leaves others in the dark of the intent.
```suggestion
            "".to_string(),
```